streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -289,6 +289,18 @@ public void testCannotCleanupWhileRunning() throws Exception {
         }
     }
 
+    @Test
+    public void testToString() {
+        streams.start();
+        String streamString = streams.toString();
+        streams.close();
+        String appId = streamString.split("\\n")[1].split(":")[1].trim();
+        Assert.assertNotEquals("streamString should not be empty", "", streamString);
+        Assert.assertNotNull("streamString should not be null", streamString);
+        Assert.assertNotEquals("streamString contains non-empty appId", "", appId);
+        Assert.assertNotNull("streamString contains non-null appId", appId);
```

Comment:
Instead of pulling the value out with a regex, what do you think of `streamsString.contains("appId")`.  Although what you have works as well.